# Fix luckybackup is not starting as superuser. Tested on KDE.
# Closes: #985169

diff -Naur luckybackup-0.5.0.orig/menu/luckybackup-pkexec luckybackup-0.5.0/menu/luckybackup-pkexec
--- luckybackup-0.5.0.orig/menu/luckybackup-pkexec	2018-11-02 15:36:19.000000000 +0100
+++ luckybackup-0.5.0/menu/luckybackup-pkexec	2021-03-18 15:23:14.183871758 +0100
@@ -1,3 +1,3 @@
 #!/bin/sh
-pkexec "/usr/bin/luckybackup" "$@"
 
+pkexec env DISPLAY=$DISPLAY XAUTHORITY=$XAUTHORITY "/usr/bin/luckybackup" "$@"
